src/doc_table.h

```diff
@@ -128,10 +112,7 @@ RSDocumentMetadata *DocTable_Put(DocTable *t, const char *s, size_t n, double sc
  * If the document ID is not in the table, the returned key's `str` member will
  * be NULL
  */
-const char *DocTable_GetKey(DocTable *t, t_docId docId, size_t *n);
-
-/* Get the score for a document from the table. Returns 0 if docId is not in the table. */
-float DocTable_GetScore(DocTable *t, t_docId docId);
+sds DocTable_GetKey(const DocTable *t, t_docId docId, size_t *n);
```

Comment:
why sds?

Reply:
It always returned sds. I just corrected the signature